Solutions/Cisco ETD/SolutionMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"publisherId": "azuresentinel",
```

Comment:
Please confirm if the solution is to be published by Microsoft. 
Haven't received any quote on publishing this Offer from our side.

If it is to be published from your end, Then change the support info

Reply:
Please change publisher id as well. It should be the one which your Partner center supports.

Reply:
The changes are made.